src/sidebar/components/PendingUpdatesNotification.tsx

```diff
@@ -17,6 +18,27 @@ export type PendingUpdatesNotificationProps = {
   clearTimeout_?: typeof clearTimeout;
 };
 
+type HideableBlockProps = {
+  hidden: boolean;
+  children: ComponentChildren;
+};
+
+/**
+ * A component that can be hidden but will be unconditionally displayed when the
+ * group it belongs to is focused or hovered
+ */
+function HideableBlock({ hidden, children }: HideableBlockProps) {
+  return (
+    <span
+      className={classnames('group-hover:inline group-focus:inline', {
+        hidden,
```

Comment:
It wasn't immediately obvious to me how this works. Reading the Tailwind docs I see that the `hidden` class expands to `display:none`. The `group-*` styles generate selectors with higher specificity that have the rule `display:inline`. These then override `hidden`.

Reply:
Yes. Do you think a comment here would be helpful.